bcipy/simulator/helpers/sim_state.py

```diff
@@ -0,0 +1,28 @@
+from abc import ABC
+from dataclasses import dataclass
+
+
+@dataclass
+class SimSessionState:
+    """ Represents the state of a current session during simulation """
+    target_symbol: str
+    current_sentence: str
+    target_sentence: str
+    display_alphabet: list[str]
+    inquiry_n: int
+    series_n: int
```

Comment:
I like the idea of capturing this all in one state object that can be passed around. We will probably want to add the lists of probabilities. Check out the `DecisionMaker` `list_series` property in task/control/handler.py.